setup.py

```diff
@@ -0,0 +1,44 @@
+from setuptools import setup, find_packages, find_namespace_packages
+import pathlib
+
+here = pathlib.Path(__file__).parent.resolve()
+
+# workaround a bug introduced by pyproject.toml
+# https://github.com/pypa/pip/issues/7953#issuecomment-645133255
+import site, sys; site.ENABLE_USER_SITE = True
+
+setup(
+  name='spinalcordtoolbox-data-<dataset>',
+  description='Part of https://github.com/neuropoly/spinalcordtoolbox',
+  long_description=(here / 'README.md').read_text(encoding='utf-8'),
+  long_description_content_type='text/markdown',
+  author='Neuropoly',
+  author_email='neuropoly@googlegroups.com',
+  url='https://spinalcordtoolbox.com/',
+  project_urls={
+      'Source': 'https://github.com/sct-data/<dataset>',
+      #'Documentation': '',
+  },
+  #license='CC-BY-NC', ??
+  #license_files=[ ... ] # TODO?
+
+  packages=find_namespace_packages('src/'),
```

Comment:
We need to use `find_namespace_packages` because each data package is nested in subdirectories, right? Maybe a comment could be good here.

Reply:
yeah this is annoying. I wish python's packaging rules weren't so arcane. I wish they were just: "here's some files, use them".

Reply:
```suggestion
  # This is weird but it makes it work
  packages=find_namespace_packages('src/'),
```